tests/threads/test_eventlet.py

```diff
@@ -2,10 +2,10 @@
 
 eventlet.monkey_patch()
 
-import threading
-import time
+import threading  #  noqa: E402
```

Comment:
use it once at head to skip all E402 violations.

Reply:
Done

Reply:
Reverted